css-forms-1/Overview.bs

```diff
@@ -829,13 +849,24 @@ ISSUE: Refine meter, progress, switch and range input styling.
   inline-size: 0;
 }
 
+input[type=checkbox][switch]::slider-fill {
+  background-color: transparent;
+  border-radius: inherit;
+}
+
 ::slider-thumb {
   border-radius: 0;
   border: none;
   background-color: currentColor;
   appearance: none;
   width: 1em;
-  height: 100%;
+  height: 1em;
```

Comment:
Need to consider the WCAG touch size requirements for this. I suspect we'll need something bigger.